apps/nrm/fields/index.js

```diff
@@ -53,27 +53,30 @@ module.exports = {
   'local-authority-contacted-about-child-local-authority-name': {
     mixin: 'select',
     validate: ['required', {type: 'maxlength', arguments: [15000]}],
-    className: ['typeahead', 'js-hidden'],
+    className: ['typeahead', 'js-hidden', 'govuk-input', 'govuk-input--error'],
```

Comment:
why is the govuk-input--error needed?

Reply:
Removed the `govuk-input--error class` and tested.